react.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const baseConfig = require(".");
+
+const eslintReactRecommendedRules = {
+  "react/display-name": 2,
+  "react/jsx-key": 2,
+  "react/jsx-no-comment-textnodes": 2,
+  "react/jsx-no-duplicate-props": 2,
+  "react/jsx-no-target-blank": 2,
+  "react/jsx-no-undef": 2,
+  "react/jsx-uses-react": 2,
+  "react/jsx-uses-vars": 2,
+  "react/no-children-prop": 2,
+  "react/no-danger-with-children": 2,
+  "react/no-deprecated": 2,
+  "react/no-direct-mutation-state": 2,
+  "react/no-find-dom-node": 2,
+  "react/no-is-mounted": 2,
+  "react/no-render-return-value": 2,
+  "react/no-string-refs": 2,
+  "react/no-unescaped-entities": 2,
+  "react/no-unknown-property": 2,
+  "react/no-unsafe": 0,
+  "react/react-in-jsx-scope": 2,
+  "react/require-render-return": 2,
+};
+
+module.exports = {
+  ...baseConfig,
+  overrides: [
```

Comment:
flyttande in alla react regler i en override så att dom inte körs i onödan på saker som inte är jsx